codec/amino.go

```diff
@@ -32,10 +32,6 @@ func New() *LegacyAmino {
 func RegisterEvidences(cdc *LegacyAmino) {
 	cdc.Amino.RegisterInterface((*tmtypes.Evidence)(nil), nil)
 	cdc.Amino.RegisterConcrete(&tmtypes.DuplicateVoteEvidence{}, "tendermint/DuplicateVoteEvidence", nil)
-	cdc.Amino.RegisterConcrete(&tmtypes.ConflictingHeadersEvidence{}, "tendermint/ConflictingHeadersEvidence", nil)
```

Comment:
Oh, I forgot this is registered here. We may want to add a note somewhere, so we don't forget to add it here once the type comes.

Reply:
Should I add a comment here? Or comment this line out instead of removing?

Reply:
I think adding a check box to the tendermint tracking issue should be fine.

Reply:
which one? This? https://github.com/tendermint/tendermint/issues/5288

Reply:
(PS: I don't have rights to edit tendermint issues)

Reply:
I added a link to this thread in the Tendermint meta-issue original comment

Reply:
BTW, for the name shouldn't we combine a string with the enum value? Example: 

"tendermint/DuplicateVoteEvidence" -> "tendermint/evidence/"+abci.EvidenceType_DUPLICATE_VOTE